internal/cmd/asyncapi/account_details.go

```diff
@@ -74,24 +74,36 @@ func (d *accountDetails) getSchemaDetails() error {
 	if err != nil {
 		return err
 	}
-	var unmarshalledSchema map[string]any
+	d.channelDetails.schema = &schema
+	unmarshalledSchema := make(map[string]any)
 	if schema.SchemaType == "" {
-		d.channelDetails.contentType = "application/avro"
-	} else if schema.SchemaType == "JSON" {
+		schema.SchemaType = "AVRO"
+	}
```

Comment:
Shouldn't we do this by default when we get the schema? (Do this inside of `GetSchemaByVersion()`)

Reply:
Refer to this thread: https://confluent.slack.com/archives/CM6PEBEGM/p1677785572528619?thread_ts=1677785042.749209&cid=CM6PEBEGM